logtail/logtail.go

```diff
@@ -136,6 +162,11 @@ type Logger struct {
 	explainedRaw   bool
 	metricsDelta   func() string // or nil
 	privateID      PrivateID
+	writeLock      sync.Mutex
```

Comment:
say what this guards.

oh, you wrote it below.

the convention is immutable stuff at the top and then blocks of:

```go

   mu sync.Mutex // guards following
   foo T
  
```